config/20-pbs.ini

```diff
@@ -25,3 +25,7 @@ accounting_log_directory = UNAVAILABLE
 ; This setting is optional and should point to your PBS server node if it is
 ; different from your OSG CE 
 pbs_server = UNAVAILABLE
+
+; The distribution of PBS you are using, either "torque" for Torque
+; or "pro" for PBS Pro.
```

Comment:
We should tell people that this is optional or that the default is set to `torque`

Reply:
I could do that but the common convention is just to have the default as a commented-out line below the text.

Reply:
Alright, go ahead and merge then